src/scene/text/canvas/utils/getCanvasFillStyle.ts

```diff
@@ -14,7 +14,7 @@ export function getCanvasFillStyle(
 {
     if (fillStyle.texture === Texture.WHITE && !fillStyle.fill)
     {
-        return Color.shared.setValue(fillStyle.color).toHex();
+        return Color.shared.setValue(fillStyle.color).setAlpha(fillStyle.alpha ?? 1).toHexa();
```

Comment:
From types, If there is HEXA `fillStyle.color` but no `fillStyle.alpha`, isn't opacity always will be `1`?